SPECS/prometheus/0001-Fixed-TestChunkDiskMapper_WriteChunk_Chunk_IterateCh.patch

```diff
@@ -1,77 +0,0 @@
-From 8bf7bc68f1d678ea82c392f80486a62e7973720b Mon Sep 17 00:00:00 2001
-From: Bartlomiej Plotka <bwplotka@gmail.com>
-Date: Thu, 25 Feb 2021 10:08:12 +0100
-Subject: [PATCH] Fixed TestChunkDiskMapper_WriteChunk_Chunk_IterateChunks for
- go1.16 (#8538)
-
-Fixes https://github.com/prometheus/prometheus/issues/8403
-
-Backported to v2.24.1 by mfrw from v2.25.0
-
-Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>
-Signed-off-by: Muhammad Falak Wani <mwani@microsoft.com>
----
- tsdb/chunks/head_chunks.go      |  4 +++-
- tsdb/chunks/head_chunks_test.go | 11 ++---------
- 2 files changed, 5 insertions(+), 10 deletions(-)
-
-diff --git a/tsdb/chunks/head_chunks.go b/tsdb/chunks/head_chunks.go
-index 051b9b1a8..d5386f7ea 100644
---- a/tsdb/chunks/head_chunks.go
```

Comment:
I was wondering if we need this patch or not ?
This patch was backported from `v2.25` onto `v2.24.1`.
The current version `v2.36.x` should not need this patch :)

Reply:
Yup, I've noticed the changes made by the patch were already in the code, so that's why I removed it.